### ISSUE TITLE
Algorithm does not favor Twitter blue users

### ISSUE THREAD

**Issue by steevjames** [twitter/the-algorithm#1681]
Algorithm does not favor Twitter blue users.

This is injustice to paid users. Elon Musk asked people to subscribe to Twitter Blue saying that they would get more priority in tweet ranking and now they don't get priorities in tweet ranking. Need increased visibility to Twitter Blue users.

**Comment by hansonhqma** [comment]
![image](https://user-images.githubusercontent.com/9426624/230201956-c5c93f6c-f59c-4385-9074-509f77080323.png)

**Comment by walksanatora** [comment]
It does though...
Atleast I thought
Since Blue replaced the verified status

**Comment by 170031284** [comment]
Basically it keep on recommending the content depending on the upvotes or like it gets for a post. 
supose if a bluestar person post a tweet and lot of his fans like it shares to other and comment on it which is making it highly engaging. It is the sign of  algorithm automatically recommends the post to people depending the information of previous people who are sharing the post

**Comment by m-a-sch** [comment]
Not all marketing claims are true. Now is a good time to think about why you wanted to identify with this verified or check-marked person and pay 8$/month for that.

**Comment by HenrikEriksson1** [comment]
POOOOOP

- [ ] Poopie

**Comment by Angus1811** [comment]
> Algorithm does not favor Twitter blue users.
> 
> This is injustice to paid users. Elon Musk asked people to subscribe to Twitter Blue saying that they would get more priority in tweet ranking and now they don't get priorities in tweet ranking. Need increased visibility to Twitter Blue users.

yes the new free speech twitter should only benefit the people who pay for free speech 

they need to add this feature please

**Comment by Angus1811** [comment]
![2c7d2e72xvra1](https://user-images.githubusercontent.com/102298861/230356161-ab96aa59-0b14-4463-b727-27377e4f5485.png)

**Comment by TolstoyDotCom** [comment]
Some Paid Blue Checks ("PBC") seem to rise to the top. However, I've seen a lot of PBCs in the "LowQuality" and "AbusiveQuality" sections. Depending on the claims Musk has made, hopefully that would result in a class action lawsuit.

However - like almost all other users - PBCs probably have no idea they're being censored so Musk might get away with it.

**Comment by TolstoyDotCom** [comment]
ChatGPT admits it isn't up to date and isn't a credible source.

It didn't take long to find an instance of a PBC being censored, see the reply from "Dan Tanner" on FoxNews/status/1646566097875763218. His tweet ("21 year old, national guard member had top security clearance from the Pentagon? A fall guy who of course will be a Trump supporter to fit the narrative..Whole stories BS..") was put in the "LowQuality" section.

If Musk's opponents were on the ball, they'd try to agitate PBCs over this.
